src/service/chat.service.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-return-await */
+import { ApiError } from '../exceptions/api.error.js';
+import { Messages } from '../modules/Messages.js';
+import { Room } from '../modules/Room.js';
+import { Token } from '../modules/Token.js';
+import { User } from '../modules/User.js';
+import { Console } from '../utils/Console.js';
+
+async function createRoom(name) {
+  const exitRoom = await Room.findOne({ where: { name } });
```

Comment:
The variable name `exitRoom` is likely a typo. Consider renaming it to `existingRoom` for clarity.